app/graphql/mutations/create_user_car.rb

```diff
@@ -20,8 +21,17 @@ def resolve(args)
         mpg: args[:mpg],
         fuel_type: args[:fuel_type]
       )
-      {car: car}
+      if car.save
+        {
+          created_car: car,
+          errors: []
+        }
+      else
+        {
+          car: nil,
+          errors: car.errors.full_messages
```

Comment:
Great job! We should talk about how we want to handle this in the front end at some point.